model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/hibernate/jsonb/JsonbType.java

```diff
@@ -105,7 +105,8 @@ public class JsonbType extends AbstractSingleColumnStandardBasicType<Object> imp
                     .addAbstractTypeMapping(MapWebAuthnPolicyEntity.class, MapWebAuthnPolicyEntityImpl.class))
             .addMixIn(UpdatableEntity.class, IgnoreUpdatedMixIn.class)
             .addMixIn(DeepCloner.class, IgnoredTypeMixIn.class)
-            .addMixIn(EntityWithAttributes.class, IgnoredMetadataFieldsMixIn.class);
+            .addMixIn(EntityWithAttributes.class, IgnoredMetadataFieldsMixIn.class)
+            .addMixIn(EnumWithUnchangeableIndex.class, EnumsMixIn.class);
```

Comment:
Nit: Could you please put the `;` to the new line so that next time a change in this part would only update a single line?